tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -509,6 +510,30 @@ public function iShouldSeeMyNameInTheTaskSRowOfTheSprintBacklog()
 	}
 
 	/**
+	 * @Then I should see the task with :points story points
+	 */
+	public function iShouldSeeTheTaskWithStoryPoints($storyPoints)
+	{
+		$this->assertElementContains('#t' . $this->selectedTask['id'] . ' .points', $storyPoints);
+	}
+
+	/**
+	 * @Then I should see the task with :title as title
+	 */
+	public function iShouldSeeTheTaskWithTitle($title)
+	{
+		$this->assertElementContains('#t' . $this->selectedTask['id'] . ' .title', $title);
+	}
+
+	/**
+	 * @Then I should see the task with :priority as priority
+	 */
+	public function iShouldSeeTheTaskWithPriority($priority)
+	{
+		$this->assertElementContains('#t' . $this->selectedTask['id'] . ' .filter-backlog', $priority);
```

Comment:
didn't you add `priority` class to the relevant `td` element to get priority here? This then should rather use `.priority` instead of `.filter-backlog`